sdk/monitor/monitor-ingestion/README.md

```diff
@@ -0,0 +1,107 @@
+# Azure Monitor Ingestion client library for JavaScript
```

Comment:
Seems like this is still the default README, would you mind tweaking it to have some minimal monitor content?

Reply:
fixed